google-cloud-datastore/src/main/java/com/google/cloud/datastore/spi/v1/HttpDatastoreRpc.java

```diff
@@ -75,12 +77,18 @@ public HttpDatastoreRpc(DatastoreOptions options) {
 
   private HttpRequestInitializer getHttpRequestInitializer(
       final DatastoreOptions options, HttpTransportOptions httpTransportOptions) {
-    final HttpRequestInitializer delegate = httpTransportOptions.getHttpRequestInitializer(options);
+    HttpRequestInitializer delegate = httpTransportOptions.getHttpRequestInitializer(options);
+    // Open Census initialization
+    CensusHttpModule censusHttpModule =
+        new CensusHttpModule(TraceUtil.getInstance().getTracer(), true);
+    delegate = censusHttpModule.getHttpRequestInitializer(delegate);
```

Comment:
try to avoid reassigning local variables

Reply:
Done